recipes/conversation_view/feed_item_row/feed_item_row_default.story.vue

```diff
@@ -18,69 +18,141 @@
+            </div>
+          </dt-stack>
+        </dt-stack>
       </template>
       <template
-        v-if="reactions"
         #reactions
       >
-        <dt-emoji-text-wrapper
-          class="d-btn d-btn--outlined d-btn--xs"
-          element-type="button"
-          size="200"
-        >
-          <span v-html="reactions" /> :smile:
-        </dt-emoji-text-wrapper>
+        <dt-recipe-emoji-row
+          :reactions="mockReactions"
+        />
       </template>
-      <template
-        v-if="menu"
-        #menu
-      >
-        <dt-icon :name="menu" />
+      <template #menu>
+        <!-- TODO replace this with DT menu -->
+        <div
+          class="d-d-flex d-bgc-primary d-bar-pill d-bc-default d-ba"
+          role="group"
+        >
+          <dt-button
+            v-for="button of hoverButtons"
+            :key="button"
+            kind="muted"
+            importance="clear"
+            size="xs"
+          >
+            <template #icon>
+              <slot name="icon">
+                <dt-icon
+                  :name="button"
+                  size="300"
+                />
+              </slot>
+            </template>
+          </dt-button>
+        </div>
       </template>
     </dt-recipe-feed-item-row>
   </ul>
 </template>
 
 <script>
 import DtRecipeFeedItemRow from './feed_item_row.vue';
-import { DtEmojiTextWrapper } from '@/components/emoji_text_wrapper';
+import { DtRecipeEmojiRow } from '../emoji_row';
 import { DtAvatar } from '@/components/avatar';
 import { DtIcon } from '@/components/icon';
+import { DtButton } from '@/components/button';
+import { DtStack } from '@/components/stack';
 
 export default {
   name: 'DtRecipeFeedItemRowDefault',
   components: {
     DtAvatar,
     DtRecipeFeedItemRow,
-    DtEmojiTextWrapper,
+    DtRecipeEmojiRow,
     DtIcon,
+    DtButton,
+    DtStack,
+  },
+
+  data () {
+    return {
+      mockReactions: [
+        {
+          emojiUnicodeOrShortname: '😀',
+          isSelected: true,
+          ariaLabel: 'Emoji aria label',
+          tooltip: 'You reacted with 😀',
+          num: 1,
+        },
+        {
+          emojiUnicodeOrShortname: '😒',
+          isSelected: false,
+          ariaLabel: 'Emoji aria label',
+          tooltip: 'You reacted with 😒',
+          num: 1,
+        },
+        {
+          emojiUnicodeOrShortname: '🙃',
+          isSelected: false,
+          ariaLabel: 'Emoji aria label',
+          tooltip: 'You reacted with 🙃',
+          num: 99,
+        },
+      ],
+
+      hoverButtons: ['bell', 'living-thing', 'map-pin'],
+
+      persons: ['Jim Halpert', 'Michael Scott', 'Pam'],
+    };
   },
 };
 </script>
+
+<style lang="less">
+  .feed-item-row__default_story {
+    p {
+      color: var(--color-foreground-primary, #000);
+      font-size: 15px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 24px;
+    }
```

Comment:
Hmmm shouldn't these styles be part of the feed item row component itself? Threads is fine to put here because that component hasn't been implemented yet

Reply:
Hmmm but the content is a slot.

I was going to suggest slot or text, but even for plain text we need to go through it and add inline rich media where needed...

Reply:
I see what you're saying since it's a slot.. but does it make sense to set this on the wrapper since MOST of the time it's going to be text to set these as defaults, which could be overridden by an element in the slot if needed? The answer may be no, I'm not totally sure how crazy this slot content is intended to get.

Reply:
Like in reality most of the feed items we see will have text with P, but in code I think it's actually the least common?
We'll probably see more components use this and put DtFeedItemPill in there with various content (meetings/call/etc)

It may make sense to have some defined CSS class from dialpad.design that defines what feed item p text should be

Reply:
> It may make sense to have some defined CSS class from dialpad.design that defines what feed item p text should be

Yeah I would agree. anyways this is good for the purposes of this PR.